wandb/sdk/lib/run_moment.py

```diff
@@ -0,0 +1,74 @@
+import typing
```

Comment:
would you mind using the
`from typing import ... ` ?
it is just the style we use in the rest of the repo